### DISCUSSION TITLE
Nginx Unit Docker Image Exposes Ports 80 and 443 alongside 8080 and 8443?

### DISCUSSION THREAD

**Discussion by jpangborn** [serversideup/docker-php#349] [Q&A]
I'm testing the `serversideup/php:8.3-unit` image on a server behind Traefik and am having trouble getting everything to work. The Ngnix Unit image is building successfully and responding on port 8080 for Laravel 11 health check. But Traefik seems to be discovering port 80 instead of 8080 for routing traffic to the service. 

In looking at the Dockerfile for the Nginx Unit variation (`src/variations/unit/Dockerfile`), I see that it exposes 80 and 443 on line 151 and exposes 8080 and 8443 on line 163. Is this correct? It's different from the Nginx FPM Dockerfile (`src/variations/fpm-nginx/Dockerfile`), which only exposes 8080 and 8443 (line 173). It also doesn't seem to match the documentation listed on https://serversideup.net/open-source/docker-php/docs/getting-started/default-configurations.

**Comment by jaydrogers** [comment]
I think this is an accident. Let me look into this and I will get back to you 👍

**Comment by jaydrogers** [comment]
## Update:
Tracking here 🤓

https://github.com/serversideup/docker-php/issues/355
